### DISCUSSION TITLE
osu! Android icon

### DISCUSSION THREAD

**Discussion by agatemosu** [ppy/osu#21455] [Ideas] [answered]
The osu! logo in Android it does not have the game modes in the background, but the application in iOS does. Having a white background instead of game modes looks really bad.

Also, it does not have a themed icon (monochrome icon).

**Comment by peppy** [comment]
Can you provide a screenshot of what it looks like to you, for reference?

**Reply by agatemosu** [reply]
This is the icon without having the rounded icons
![osu! Android icon](https://user-images.githubusercontent.com/58957468/204819234-410204eb-f10c-4160-ad01-72ea4f57378a.png)
This is comparing it with other themed icons
![Themed Icons](https://user-images.githubusercontent.com/58957468/204818911-0e047d7b-638c-4204-9dbd-b68a888a7e50.png)

**Reply by ghost** [reply]
are you using some sort of icon pack or are these the default icons for other apps

**Reply by agatemosu** [reply]
the first one is one ui launcher and the second one is pixel launcher

**Reply by peppy** [reply]
Thanks. I think this should be fixed with proper updated assets. If someone is interested in looking into this and requires assets let me know.

**Reply by bdach** [reply]
I could probably give it a go. Assets would be needed, though, since to make it work layers have to be split and sizings have to be adjusted for this to work as expected (https://developer.android.com/develop/ui/views/launch/icon_design_adaptive for more info).

**Reply by Joehuu** [reply, marked as answer]
Will be addressed in https://github.com/ppy/osu/pull/26704.
